spyder/widgets/github/gh_login.py

```diff
@@ -53,31 +55,63 @@ def __init__(self, parent, username):
         basic_form_layout = QFormLayout()
         basic_form_layout.setContentsMargins(-1, 0, -1, -1)
 
+        basic_lbl_msg = QLabel(_("For regular users, i.e. users <b>without</b>"
+                                 " two-factor authentication enabled"))
+        basic_lbl_msg.setWordWrap(True)
+        basic_lbl_msg.setAlignment(Qt.AlignJustify)
+
         lbl_user = QLabel(_("Username:"))
         basic_form_layout.setWidget(0, QFormLayout.LabelRole, lbl_user)
         self.le_user = QLineEdit()
         self.le_user.textChanged.connect(self.update_btn_state)
         basic_form_layout.setWidget(0, QFormLayout.FieldRole, self.le_user)
+        basic_form_layout.addRow("", QWidget())
 
         lbl_password = QLabel(_("Password: "))
-        basic_form_layout.setWidget(1, QFormLayout.LabelRole, lbl_password)
+        basic_form_layout.setWidget(2, QFormLayout.LabelRole, lbl_password)
         self.le_password = QLineEdit()
         self.le_password.setEchoMode(QLineEdit.Password)
         self.le_password.textChanged.connect(self.update_btn_state)
-        basic_form_layout.setWidget(1, QFormLayout.FieldRole, self.le_password)
+        basic_form_layout.setWidget(2, QFormLayout.FieldRole, self.le_password)
+
+        self.cb_remember = None
+        # Check if we are in Python 2 and Linux where keyring needs a backend
```

Comment:
Please change this comment to

    # Check if we are not in Python 2 and Linux because there's no keyring backend there